resources/iso_eventTypeCode.yml

```diff
@@ -0,0 +1,25 @@
+---
+# mdJson codelists
+# base codelist for ISO 19115-2, 19115-1
+
+codelistType: "staticList"
+codelistName: "iso_eventTypeCode"
+source: "ISO"
+sourceName: "MI_EventTypeCode"
+extensible: true
+description: "code indicating whether the data contained in this packet is real (originates from live-fly or other non-simulated operational sources), simulated (originates from target simulator sources), or synthesized (a mix of real and simulated data)."
```

Comment:
@dwalt this description doesn't seem match the codelist. Can you verify. According to https://schemas.isotc211.org/schemas/Resources/codelists.xml the definition is "type of event related to platform/instrument/sensor"

Reply:
@dwalt @hmaier-fws That description was not sourced from anything official. It has been changed to reflect the correct description.